pitop/processing/utils/vision_functions.py

```diff
@@ -67,8 +72,56 @@ def find_largest_rectangle(rectangles):
 
 
 def scale_frame(frame, scale):
-    cv2 = import_opencv()
     scaled_width = int(frame.shape[1] * scale)
     scaled_height = int(frame.shape[0] * scale)
     dim = (scaled_width, scaled_height)
     return cv2.resize(frame, dim, interpolation=cv2.INTER_AREA)
+
+
+def resize(image, width=None, height=None, inter=cv2.INTER_AREA):
```

Comment:
@jcapona do we really want to import opencv every time this function is called? I don't mind doing this for single-call function during initialization but it seems like it will bring a massive overhead when doing it 30-60 times per second in a frame callback

Reply:
I read somewhere that having the cv2.INTER_AREA in the parameter itself improves efficiency, I'll dig it up - might not be necessary for us but interesting nonetheless

Reply:
```
pi@pi-top:~ $ time python3 -c "import cv2"

real	0m2.902s
user	0m2.706s
sys	0m0.181s
pi@pi-top:~ $ time python3 -c "import cv2; import cv2"

real	0m2.926s
user	0m2.741s
sys	0m0.180s
pi@pi-top:~ $
```

Seems pretty low cost to perform additional imports.